src/components/__tests__/InfiniteHits.js

```diff
@@ -107,6 +128,21 @@ it('renders correctly with a custom item rendering', () => {
   expect(wrapper.html()).toMatchSnapshot();
 });
 
+it('renders correctly on the first page', () => {
```

Comment:
We could add a test when it's not the first page but enabled.

Reply:
@yannickcr You forgot to apply the feedback?

Reply:
No, I added a ["renders correctly when not on the first page"](https://github.com/algolia/vue-instantsearch/pull/659/commits/78e9094bc5d4e67ae652ec02ce117c3d5c443215) test :wink: